source/content/guides/drupal-9-from-community.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Create a Drupal 9 Site Using a Community Distribution
+description: Learn how to create a site using a Drupal 9 community distribution.
+cms: "Drupal"
+categories: [get-started]
+tags: [site]
+type: guide
+permalink: docs/guides/drupal-9-from-community
+contributors: [wordsmither]
+date: 07/06/2022
+---
+
+Distributions are pre-made packages that you can use to simplify creating and setting up a Drupal website. Drupal distributions are exceptionally helpful if you want to create a website, but don't want to build it from scratch. 
+
+Start by following the documentation provided with the community distribution; then continue with the following steps.
+
+## Requirements
+
+- You are managing site dependencies using Composer.  If not, you can [convert a standard Drupal site to a Composer managed site](/guides/composer-convert).
+
+## Create an Empty Upstream
+
+There are two ways to create an empty Upstream site: via the [Pantheon Dashboard](/create-sites) and via [Terminus](/terminus).
+
+- Via the Pantheon Dashboard:
+
+  - Use the [Empty Site Upstream](https://dashboard.pantheon.io/sites/create?upstream_id=4c7176de-e079-eed1-154d-44d5a9945b65)
+
+- Via Terminus:
+
+  ```bash{promptUser: user}
+  terminus site:create my-new-site "My New Site" empty
+  ```
+
+## Create the Community Distribution Project 
+
+Use the documentation provided with the community distribution to run the recommended Composer `create-project` command.  Here are some examples
+
+### Apigee
+
+  ```bash{promptUser: user}
+  composer create-project apigee/devportal-kickstart-project:9.x-dev MY_PROJECT --no-interaction
+  ```
+
+### Drupal Thunder
+
+  ```bash{promptUser: user}
+  composer create-project thunder/thunder-project thunder --no-interaction --no-install
+  ```
+
+### DKAN Open Data Platform
+
+  ```bash{promptUser: user}
+  composer create-project getdkan/recommended-project my-project
+  ```
+
+<Alert title="Warning" type="danger" >
+
+Do not rely on the sample code above.  Be sure to use the instructions provided with the community distribution.
```

Comment:
"Refer to the distro documentation to get the right composer create-project command" or similar. Maybe an info instead of warning? And I'd remove the first part: "Do not rely..."